psdk_wrapper/src/modules/perception.cpp

```diff
@@ -0,0 +1,404 @@
+    case 5:
+      RCLCPP_INFO(get_logger(), "Subscribe right stereo camera pair images.");
+      return_code = DjiPerception_SubscribePerceptionImage(
+          DJI_PERCEPTION_RECTIFY_RIGHT, &c_DjiTest_PerceptionImageCallback);
+      break;
+  }
+
+  if (return_code != DJI_ERROR_SYSTEM_MODULE_CODE_SUCCESS)
+  {
+    RCLCPP_ERROR(get_logger(),
+                "Could not start the perception stereo vision stream. Error code: %ld",
+                return_code);
+    return false;
+  }
+  RCLCPP_INFO(get_logger(), "Perception stereo cameras stream started successfully...");
+  return true;
+}
+
+bool
+PerceptionModule::stop_perception_stereo_cameras_stream(
+    const uint stereo_cameras_direction)
+{
+  T_DjiReturnCode return_code;
+  switch (stereo_cameras_direction)
+  {
+    case 0:
+      RCLCPP_INFO(get_logger(), "Unsubscribe down stereo camera pair images.");
+      return_code =
+          DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_DOWN);
+      break;
+    case 1:
+      RCLCPP_INFO(get_logger(), "Unsubscribe front stereo camera pair images.");
+      return_code = DjiPerception_UnsubscribePerceptionImage(
+          DJI_PERCEPTION_RECTIFY_FRONT);
+      break;
+    case 2:
+      RCLCPP_INFO(get_logger(), "Unsubscribe rear stereo camera pair images.");
+      return_code =
+          DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_REAR);
+      break;
+    case 3:
+      RCLCPP_INFO(get_logger(), "Unsubscribe up stereo camera pair images.");
+      return_code =
+          DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_UP);
+      break;
+    case 4:
+      RCLCPP_INFO(get_logger(), "Unsubscribe left stereo camera pair images.");
+      return_code =
+          DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_LEFT);
+      break;
+    case 5:
+      RCLCPP_INFO(get_logger(), "Unsubscribe right stereo camera pair images.");
+      return_code = DjiPerception_UnsubscribePerceptionImage(
+          DJI_PERCEPTION_RECTIFY_RIGHT);
+      break;
+  }
+
+  if (return_code != DJI_ERROR_SYSTEM_MODULE_CODE_SUCCESS)
+  {
+    RCLCPP_ERROR(get_logger(),
+                "Could not stop the perception stereo vision stream. Error code: %ld",
+                return_code);
+    return false;
+  }
+  RCLCPP_INFO(get_logger(), "Perception stereo cameras stream stoped successfully...");
+  return true;
+}
+
+// To clear the previous stereo camera stream this function is used.
+void
+PerceptionModule::clear_perception_stereo_cameras_stream()
+{
+  DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_DOWN);
+  DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_FRONT);
+  DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_REAR);
+  DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_UP);
+  DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_LEFT);
+  DjiPerception_UnsubscribePerceptionImage(DJI_PERCEPTION_RECTIFY_RIGHT);
+}
```

Comment:
This method could be optimize. Consider storing the different options in a global variable of some type of vector and use a for loop to unsubscribe from each one of them. Moreover, you could return a boolean indicating the success of the operation